app/src/main/java/edu/outside2154/gamesense/model/Stat.kt

```diff
@@ -1,6 +1,11 @@
 package edu.outside2154.gamesense.model
 
-class Stat (initGoals: Map<String, Double>) {
+import com.google.firebase.database.DataSnapshot
+import com.google.firebase.database.DatabaseReference
+import edu.outside2154.gamesense.database.FirebaseTransform
+
+class Stat (initGoals : Map<String, Double>, currGoals : Map<String, Double>)
+    : FirebaseTransform<Stat> {
 
     data class StatItems(val items: Map<String, Double>) {
```

Comment:
I just realized it would be better if we extended `Map<String, Double>` to avoid indirection through `items`. I can do it in a later PR.